scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -144,30 +170,6 @@ export function RelatedItemsDirective(authoringWorkspace: AuthoringWorkspaceServ
             };
             scope.refreshRelatedItems();
 
-            /**
-             * Reorder related items on related items list
-             *
-             * @param {int} start
-             * @param {int} end
-             */
-            scope.reorder = (start, end) => {
```

Comment:
was this unused or did you remove the feature in this PR?

Reply:
oops, I don't remember why I deleted this but it's not related to my PR, it should be there.